drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -594,9 +632,11 @@ function DragAndDropBlock(runtime, element, configuration) {
     DragAndDropBlock.STANDARD_MODE = 'standard';
     DragAndDropBlock.ASSESSMENT_MODE = 'assessment';
 
+    var MEDIA_QUERY = '(max-width: 1001px)';
```

Comment:
I don't think we have to differentiate between mobile and desktop in JS at this point. I would prefer to only use media queries in CSS.